terraform/validator/main.tf

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The validator modules will be shared across platform. Therefore, everytime we add a new platform such as `eks ec2`, we only need to
```
module "validator" {
  source = "../validator"

  arc                   = var.arc
  family               = "linux"
  action              = "upload"
  s3_bucket       = var.s3_bucket
  test_dir            = var.test_dir
}
```